index.css

```diff
@@ -7,51 +7,61 @@ body {
  font-style: normal;
  margin: 0;
  padding: 0;
-}
-.all-content-wrapper {
-    max-width: 90%;
-    margin: 0 auto;
-    padding: 0 16px;
+ display: flex;
+flex-direction: column;
 }
 
 /*Navbar*/
 
 .navbar {
     display: flex;
     justify-content: space-between;
-    max-width: 100%;
-    margin: 30px 0px 30px 0px;
+    max-width: 100vw;
+    padding: 1% 5% 1% 5%;
     align-items: center;
+    text-transform: uppercase;
+    background-color: #1F1F1F;
+    position: fixed;
```

Comment:
I think there is something going on between the display of flex, and position sticky.   I played with some variations but didn't get what the right combo is.   I can see via inspect that there is overlap of the boxes. Some of my changes put the navbar all the way to the left.   Rio Edwards is very good with flexbox - would suggest you setup session with him.  I would love to find out what it is though.